src/main/resources/templates/addCustomerForm.html

```diff
@@ -42,6 +42,10 @@ <h1>Add a new Customer</h1>
 </div>
 
 <script src="js/carousel.js"></script>
+
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
```

Comment:
Suggestion for the future, since these are common to all pages, perhaps worth looking into Thymeleaf layouts https://www.thymeleaf.org/doc/articles/layouts.html